react/features/base/conference/middleware.any.ts

```diff
@@ -295,7 +297,14 @@ function _conferenceJoined({ dispatch, getState }: IStore, next: Function, actio
     // handles the process of leaving the conference. This is temporary solution
     // that should cover the described use case as part of the effort to
     // implement the conferenceWillLeave action for web.
-    beforeUnloadHandler = () => {
+    beforeUnloadHandler = (e?: any) => {
+        if (LocalRecordingManager.isRecordingLocally()) {
+            dispatch(stopLocalVideoRecording());
+            if (e) {
+                e.preventDefault();
```

Comment:
Won't this prevent the other event handlers from sunning? LJM has a beforeunload handler that closes the connection.

Reply:
Not sure, but I don't think so. I think `stopPropagation` would do that

Reply:
Right indeed!